linter.py

```diff
@@ -231,11 +232,10 @@ def _uncovered_to_tuple(self, uncovered, uncovered_lines):
         Since flow produces JSON output, there is no need to match error
         messages against regular expressions.
         """
-        match = self.filename == uncovered.get('source')
```

Comment:
Why remove that line? As I read this it should remove errors coming from other files.

Reply:
You're probably right that I should add this back. It wasn't clear to me how the source could be other files (and it didn't come up in my local testing).